mautic/mautic
# Prevented Froala content from getting trapped and improved drag/drop with scrolling

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) |  #3129
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR fixes a bug where occasionally (hard to reproduce), the froala quick insert and other froala elements would get stuck in the rendered HTML. 

It also tries to improve drag/drop support for scrolling and make it easier to move slots.

It updates Froala to fix some scroll issues.

#### Steps to test this PR:
1. Use the builders in page/email and ensure they continue to function properly. Note that the editor is no longer inline but populates a content textarea in the right.
